src/paths.ts

```diff
@@ -8,7 +8,17 @@ export function getCommunicationDirPath() {
   // bother with a suffix
   const suffix = info.uid >= 0 ? `-${info.uid}` : "";
 
-  return join(tmpdir(), `vscode-command-server${suffix}`);
+  // NB: See https://github.com/talonhub/community/issues/966 for why we do
+  // per-os directories
+  if (process.platform === "win32") {
+    return join(`${homedir()}\\AppData\\Roaming\\talon\\`, `vscode-command-server${suffix}`);
```

Comment:
note that you don't use the suffix for windows talon side. Probably doesn't matter because it will be empty on windows, but we should be consistent